stdlib/public/SDK/Foundation/Decimal.swift

```diff
@@ -13,6 +13,8 @@
 @_exported import Foundation // Clang module
 import _SwiftCoreFoundationOverlayShims
 
+// The methods in this extension exist to match the protocol requirements of FloatinPoint, even if we can't conform directly.
```

Comment:
Typo: "FloatinPoint"

Reply:
Alternatively: we propose to amend IEEE-754 for Floatin' Point numbers

Reply:
(I'll wait for the full macOS test to pass before amending so we can make sure it's green; I'll smoke test afterwards)